common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -264,6 +264,7 @@ def setUp(self):
             thread_id
         )
         self.thread_page.visit()
+        self.sort_page = DiscussionSortPreferencePage(self.browser, self.course_id)
```

Comment:
Just a nit: This line will be called for every test under this test class. I don't think we really need it to be called without any use in other test cases.